commcare_connect/program/views.py

```diff
@@ -45,7 +39,7 @@ class ProgramList(ProgramManagerMixin, SingleTableView):
     table_class = ProgramTable
 
     def get_queryset(self):
-        return Program.objects.filter(organization=self.request.org)
+        return Program.objects.filter(organization=self.request.org).order_by("date_modified")
```

Comment:
Nit: i think start date or alphabetical is easier becaues it will be more stable and predictable for the users

Reply:
Added [0924f21]([0924f21](https://github.com/dimagi/commcare-connect/pull/385/commits/0924f21617da12af23fd9b877b3f13bc67d70a42))

Reply:
Added [0924f21](https://github.com/dimagi/commcare-connect/pull/385/commits/0924f21617da12af23fd9b877b3f13bc67d70a42)